Documentation/packaging-installation.md

```diff
@@ -0,0 +1,45 @@
+# .NET Core Packaging and Installation
+
+This document helps you install or package an SDK built using source-build.
+
+The SDK built by source-build is generally not portable. That means it will work on the same operating system where it was built. It will not work on older operating systems. It may work on newer operating systems.
+
+## Using the SDK directly
+
+The built SK is generally located at `artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz`. To install and use it, unpack it to any directory and then use the `dotnet` executable.
+
+For example, you can extract the tarball to `/opt/dotnet` and use it from there:
+
+```
+$ sudo mkdir /opt/dotnet/
+$ sudo tar xf artifacts/x64/Release/dotnet-sdk-3.1.105-fedora.32.tar.gz -C /opt/dotnet/
+$ /opt/dotnet/dotnet --info
+```
+
+## Creating a Linux distribution package
+
+If you want to create a Linux distribution package (`rpm`, `deb`) out of it, here are some suggestions.
+
+1. Extract the tarball to the distribution appropriate location such as `/usr/lib64/dotnet/`, `/usr/lib/dotnet/`, or `/usr/lib/x86_64-linux-gnu/dotnet/`
+
+2. Create a symlink from `/usr/bin/dotnet` (or equivalent) to where you installed the SDK in the previous step. This lets users simply run `dotnet` and have it work.
+
+3. Create an `/etc/dotnet/install_location` and add the location where you installed the SDK in there. This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
+
+4. Define the environment variable `DOTNET_ROOT` and make it point to where you installed the SDK. This environment variable is used instead of `/etc/dotnet/install_location` by older .NET Core versions.
+
+5. Add `~/.dotnet/tools` to the `$PATH`. Otherwise you wont be able to run dotnet tools that you install directly.
+
+There are several other optional things you can do:
+
+- .NET Core source repositories include man pages. You can search for them and package them up: `find -iname '*.1' -exec cp {} /usr/share/man/man1/ \;`
+
+- .NET Core includes bash-completion and zsh-completion scripts. You can install them.
```

Comment:
I guess it speaks to your suggestion on linking to a better resource, because that page should be replaced by https://github.com/dotnet/cli/pull/13384

Reply:
Done! Thanks for the idea.